certs/pki/am-certs/am_certs.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+package sdk
+
+import (
+	"github.com/absmach/certs/sdk"
+	"golang.org/x/crypto/ocsp"
+)
+
+type Agent interface {
+	Issue(entityId, ttl string, ipAddrs []string) (sdk.Certificate, error)
```

Comment:
The return certificate struct should be belongs to magistrala certs service. 
So please convert it magistrala certs object.